OxfordBodleian/Juel-Jensen/BDLaethb5.xml

```diff
@@ -214,65 +9135,196 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        Erroneous text is expunctuated with dashes above and
+                                        below (eg.,  <locus target="#108vc"/>). <foreign xml:lang="gez">Tamallas</foreign>-signs in the margins point to scribal corrections (eg., 
+                                        <locus target="#158v #206v"/>) or to the place in which sentences have been omitted, which have been then added
+                                        in the margin (eg., <locus target="#109va"/>). The function of all <foreign xml:lang="gez">tamallas</foreign>-signs is not always immediately
+                                        clear (eg., <locus target="#72r #150v"/>). Occasionally,  commemorations or salām-hymns have been 
+                                        added in the margins (eg., <locus target="#48vb #143v"/>).
+                                    </desc>
+                                </item>
+                                
+                                <item xml:id="e6">
+                                    <locus target="#91v"/>
+                                    <desc>A cross in black ink, possibly by a European hand, in the
+                                        upper outer corner.</desc>
+                                </item>
+                                
+                                <item xml:id="e7">
+                                    <locus target="#1r"/>
+                                    <desc> A white sticker with the name of <persName
+                                        ref="PRS5782JuelJen"/> printed (<foreign xml:lang="gez"
+                                            >ቤንት፡ ዩውል፡ የንሰን።</foreign>), and the signatures Ms.
+                                        Aeth. b. 5 and Ms. Et 32 written in pencil is glued on the page.</desc>
+                                </item>
+                            </list>
+                        </additions>
+
+                        <!--   <bindingDesc>
                             <binding xml:id="binding">
                                 <decoNote xml:id="b1"></decoNote>
                                 <decoNote xml:id="b2" type="bindingMaterial">
                            <material key="wood"/>
                         </decoNote>
                             </binding>
                         </bindingDesc>-->
-                        
+
                     </physDesc>
-                    
-                <!--    <history>
-                        <origin>
-                            <origDate when="" evidence=""/>
-                        </origin>
-                        <provenance>  </provenance>
-                    </history>-->
-                    
+
+                    <history>
+                        <origin> The colophon states that the manuscript was completed during the reign of
```

Comment:
Should we provide the colophon in text and translation since it refers to UM? I think  it is highly interesting Ms and may deserve a typed colophon with translation :)

Reply:
I've added the translation with a div